src/js/pensions/config/form.js

```diff
@@ -853,9 +830,8 @@ const formConfig = {
                 minItems: 1,
                 items: {
                   type: 'object',
-                  required: ['dependentRelationship', 'fullName'],
+                  required: ['fullName', 'childDateOfBirth'],
```

Comment:
Does adding the `childDateOfBirth` here make it required even when there are no depended children to report?